src/runtime/test/strategies/convert-constraints-to-connections-tests.js

```diff
@@ -26,7 +28,7 @@ describe('ConvertConstraintsToConnections', async () => {
       recipe
         A.b -> C.d`)).recipes[0];
     const inputParams = {generated: [{result: recipe, score: 1}]};
-    const cctc = new ConvertConstraintsToConnections({pec: {}});
```

Comment:
Can you compare with:

https://github.com/PolymerLabs/arcs/pull/2515

Reply:
done (and also added the context in a follow up commit)